contributors/devel/sig-node/triage.md

```diff
@@ -0,0 +1,153 @@
+
+## Triage
+
+All new PRs added to the board should begin in the **Triage** column.
+
+When a pull request is made against kubernetes/kubernetes, it will typically
+have the following [labels], applied by the Prow bot:
+
+- needs-triage
+- needs-priority
+- needs-ok-to-test (not needed for project members)
+
+In order to be moved out of this column, all of the above labels must be set.
+
+[labels]: https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md
+
+### needs-ok-to-test
+
+Use your best judgment in determining whether a PR is OK to test. You don't
+have to do a full review: just make sure that the code does not appear to be
+actively malicious, and the PR appears to be doing something useful. Use the
+command `/ok-to-test`.
+
+Only [Kubernetes org members] can add this label.
+
+If the PR is trivial and doesn't provide much value, feel free to close it
+using the `/close` command and link the author to the [trivial edits policy].
+
+[Kubernetes org members]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
+[trivial edits policy]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits
+
+### needs-priority
+
+You can take a quick look at what the PR is addressing and then apply a
+priority label.
+
+Anyone can add this label.
+
+- **priority/critical-urgent:** Urgent bug fix, required ASAP. If not
+  addressed, will block a release. These issues should always be discussed in
+  the `#sig-node` channel on Slack.
+- **priority/important-soon:** Needs to be completed this release. Important
+  bug fixes + KEPs targeted for the current milestone.
+- **priority/important-longterm:** Has an attached issue/KEP, but unclear what
+  the specific priority is.
+- **priority/backlog:** Non-urgent changes such as typo fixes, linters, small
+  comment fixes, etc.
+
+### needs-triage
+
+There are two more things to check before accepting a PR for triage.
+
+The first is whether the SIG is correct. If the PR does not appear to touch SIG
+Node code or require a SIG Node approver, you should remove the SIG Node label,
+and add other SIG labels as appropriate.
+
+The second is verifying the kind of PR. Most will be bug fixes, cleanups, or
+documentation. Feature PRs should generally have an attached KEP. API changes
+and deprecations require special review; those labels may be mistakenly
+applied, so check over the PR to make sure they're accurate.
+
+Once you've quickly looked over a PR, applied the appropriate labels, and it
+looks ready to proceed to review (i.e. it doesn't have any labels that would
+mean it's waiting on more work from the author), you can mark the PR as triaged
+with `/triage accepted`.
+
+Only [Kubernetes org members] can add this label.
+
+## Waiting on Author
+
+This column means that the PR is waiting on some action from the author. A
+reviewer may have requested changes, or a PR may have one of the following
```

Comment:
```suggestion
reviewer may have requested changes, tests might be failing, or a PR may have one of the following
```

Reply:
I typically do *not* put PRs into **Waiting on Author** if tests are failing because tests are rather flaky, only if there is a confirmed test failure that needs to be addressed (and usually at that point I put a hold on it).

Reply:
Maybe we should to save reviewers time. It's easier to review when you know that tests are passing. Maybe link instruction on how to investigate test failures.